packages/office-ui-fabric-react/src/components/Toggle/Toggle.styles.ts

```diff
@@ -63,6 +63,8 @@ export const getStyles = (props: IToggleStyleProps): IToggleStyles => {
       }
     ],
 
+    input: ['ms-Toggle-checkbox', hiddenContentStyle],
```

Comment:
remove 'ms-Toggle-checkbox', we don't want to add more public classes

Reply:
Updated